Documentation/devel/update-coreos-stage1.md

```diff
@@ -0,0 +1,129 @@
+# Update coreos flavor stage1
+
+This guide will guide you through updating the version of the coreos flavor of stage1.
+We usually want to do this to update the systemd version used by the stage1.
+
+The process is quite manual because it's not done often, but improvements are welcomed.
```

Comment:
it did come up as "welcomed" when I was writing it though, so I should follow my intuition :P